README.md

```diff
@@ -11,7 +11,9 @@ Unless you want to develop this package, it is recommended that you use it from
 
 1. [Enable the MELPA repository](https://melpa.org/#/getting-started).
 
-2. Add `(require 'julia-mode)` to your Emacs init file.
+2. Add `(package-install 'julia-mode)` to your Emacs init file.
+
+3. Add `(require 'julia-mode)` to your Emacs init file.
 
```

Comment:
```suggestion
2. [Install](https://www.gnu.org/software/emacs/manual/html_node/emacs/Package-Installation.html) the `julia-mode` package.

3. Add `(require 'julia-mode)` to your Emacs init file.
```

Reply:
@Volker-Weissmann looks like I misused the suggest-changes feature, and it messed up this patch. You'll have to manually remove the repeated lines. Sorry.